spec/hot.spec.js

```diff
@@ -81,8 +85,12 @@ describe('HtmlWebpackPluginHMR', () => {
         const hotUpdateJsFileNames = Object.keys(stats.compilation.assets).filter((fileName) => /\.hot-update\.js$/.test(fileName));
         expect(hotUpdateJsFileNames).not.toEqual([]);
         expect(hotUpdateJsFileNames.length).toEqual(1);
-        const hotUpdateFileSource = stats.compilation.assets[hotUpdateJsFileNames[0]].source();
-        expect(hotUpdateFileSource).not.toEqual('');
+        if (webpackMajorVersion < 5) {
```

Comment:
Why do we need this check?

What about removing the old `hotUpdateFileSource` code as you added tests based on size anyway?

Reply:
True, I'll take it out :)